src/seedDashboard/seedAbout.ts

```diff
@@ -21,6 +21,13 @@ export class SeedAbout{
     ["pdf", "fas fa-file-pdf"],
   ]);
 
+  formatLink(link: string): string {
+    const pattern = /^(?:http(s)?:\/\/)/;
```

Comment:
what will this do if they have entered "//medium.com"?  Will you end up with "////medium.com"?  

what is they entered "HTTPS://medium.com"?   Will you end up with "HTTPS:////medium.com"?  

More generally:  is there any way this can screw up?

Reply:
The user could not have entered //medium.com as the stage1 validation would flag it as invalid. Likewise, the validation on stage1 also flags HTTPS://medium.com as the regex used for the validation is not cases sensitive

Reply:
> the validation on stage1 also flags HTTPS://medium.com as the regex used for the validation is not cases sensitive

Don't understand.  The question is about what `formatLink` will do when it encounters  HTTPS://medium.com .

Reply:
it would add a // to the link because the regex isn't case sensitive so it would not pick HTTPS://